docs/dashboards/detection-response-dashboard.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[detection-response-dashboard]]
+= Detection & Response dashboard
+
+The Detection & Response dashboard provides focused visibility into the day-to-day operational activities of your security environment. Security operations managers and analysts can quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with frequent and severe alerts.
+
+[role="screenshot"]
+image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]
+
+Interact with various dashboard elements:
+
+* Use the date and time picker in the upper-right to specify a timeframe for displaying information on the dashboard. 
+
+* In sections that list alert counts, click the number to investigate the alerts in Timeline.
+
+* Click the names of detection rules, cases, hosts, and users to open their details pages.
```

Comment:
```suggestion
* Click the name of a detection rule, case, host, or user to view more details.
```

Reply:
Since you can only click one at a time

Reply:
I want to convey that clicking a name opens a new page and navigates away from the dashboard, vs. just showing more details which could imply a less disruptive tooltip or flyout. Might be overthinking how much nuance readers pick up on, but gotta try at least. 😅 Also we seem to use plural for things like "rule details page" so this follows that pattern. Maybe something like:
```suggestion
* Click the name of a detection rule, case, host, or users to open its details page.
```

Reply:
^^ 💯